itou/fixtures/postgres/cities.sql

```diff
@@ -1,3 +1,6 @@
+-- Created with command:
+-- pg_dump --no-owner --no-privileges --data-only -d itou -t cities_city > /backups/cities.sql
```

Comment:
Le `/backups/cities.sql` est spécifique à l'env docker donc je pense qu'on peux le virer ou le remplacer par le bon chemin dans le repo.
```suggestion
-- pg_dump --no-owner --no-privileges --data-only -d $(PGDATABASE) -t cities_city
```

Reply:
Yes, donc `./itou/fixtures/postgres/cities.sql`.

Reply:
J’ai un petit débat interne avec ça. Si on dit que l’environnement cible pour PostgreSQL est docker, alors on devrait exécuter la commande dans un docker, non ?

Reply:
Je vois plus PostgeSQL (et le reste) dans docker comme l'environnement par défaut qui fonctionne _out of the box_, et pas comme un environnement cible qui est plus du _one to rule them all_.
On a pas de truc bizarre ou compliqué donc autant essayer de laisser le choix que de s'imposer un fonctionnement.

Reply:
C’est ce qui me dérange. On se retrouve à devoir penser en permanence à “et si c’est dans un docker ?” ou “et si c’est dans un venv ?”, et à choisir l’un, l’autre ou les deux en cible de doc.

Le fait d’avoir plusieurs environnements cibles ajoute de la complexité, dans la mise en place de l’environnement de dev (instructions à base de si docker -> ... si venv -> ..., faire X dans docker, faire X dans venv) et dans plusieurs tâches de maintenance (ex. dans cette PR : où stocker le fichier de résultat, comment appeler `pg_dump` car un développeur utilisant docker n’a pas besoin d’avoir les binaires postgres sur sa machine hôte), et c’est une question qui revient régulièrement au fil des PRs.

J’aimerais bien avoir un _setup_ recommandé, **et** un ou plusieurs _setup_ alternatifs. De cette manière, on documente pour l’environnement recommandé, on s’assure que la doc, les scripts, et les recettes fonctionnent pour l’environnement recommandé. Pour les autres environnements, on fait au mieux pour garder la compat, mais il faut potentiellement des adaptations de la doc pour les utiliser.

Proposition :
- par défaut, documenter comme si le dev était dans un environnement local (pas de docker exec) pour les commandes, donc les executables python, postgres, et le code source sont dans le `PATH`. Ça permet une doc sans boilerplate autour des commandes, et sans copier des fichiers entre docker et l’hôte.
- la doc d’installation vise à mettre en place cet environnement, où il est le plus agréable de travailler (à mon avis)
- une doc d’installation alternative explique comment mettre en place et exploiter l’environnement docker :
    - avoir un client psql
    - lancer une management command
    - lancer les tests
    - copier un fichier d’un docker vers l’hôte
- maintenir un `make run` qui lance toute l’app (dans des dockers, pour faciliter une installation basique)

Reply:
Je suis tout à fait d'accord avec cette analyse et en particulier avec la proposition qui documente les commandes via le `PATH` et non via docker.

Ca permet de ne pas alourdir la doc, de savoir s'adapter à des environnements minimaux ou à ce qui se passe en prod (où `docker` ne tourne pas) etc.

OK pour documenter docker mais a minima, en doc alternative et ne maintenir que le `make run` en tant que commande officiellement utilisant `docker`, quitte à la renommer `make docker_run` ou quelque chose qui rappelle explicitement que c'est un mode alternatif.